source/playlunky/mod/sprite_sheet_merger_sheet_data.cpp

```diff
@@ -894,3 +894,20 @@ void SpriteSheetMerger::MakeMenuBasicTargetSheet()
         .SourceSheets{ std::move(source_sheets) },
         .MultiSourceTiles{ std::move(head_tile) } });
 }
+void SpriteSheetMerger::MakeCaveDecoTargetSheet()
+{
+    std::vector<SourceSheet> source_sheets{
+        SourceSheet{
+            .Path{ "Data/Textures/Entities/udjat_wall_heads" },
```

Comment:
Wonder if we should put this under Entities/Decorations in case we ever wanna add more.